after/ftplugin/qf.vim

```diff
@@ -265,7 +265,7 @@ endfunction
 augroup QuickrPreviewQfAutoCmds
     autocmd! * <buffer>
     " Auto close preview window when closing/deleting the qf/loc list
-    autocmd BufDelete <buffer> silent! pclose
+    autocmd WinClosed <buffer> silent! pclose
```

Comment:
Can you add a version check to make sure `BufDelete` is called for the older versions? We don't want to break it for older versions.